packages/manager/modules/dbaas-logs/src/logs/detail/aliases/link/aliases-link.controller.js

```diff
@@ -77,7 +77,7 @@ export default class LogsAliasesLinkCtrl {
 
     this.$q.all([this.alias.promise, this.streams.promise]).then((result) => {
       const diff = filter(
-        result[1].data,
+        result[1],
         (stream) =>
           stream.isEditable &&
```

Comment:
`stream.isEditable &&` on this filter is not required as `getOwnStreams()` already do it.

Reply:
done.